crates/divan_compat/macros/src/lib.rs

```diff
@@ -25,21 +31,45 @@ pub fn bench_compat(attr: TokenStream, item: TokenStream) -> TokenStream {
 
     for arg in parsed_args.args {
         match &arg {
-            Meta::NameValue(MetaNameValue { path, .. }) if path.is_ident("crate") => {
+            NameValue(MetaNameValue { path, .. }) if path.is_ident("crate") => {
                 return quote! {
-                    compile_error!("crate argument is not supported with codspeed_divan_compat");
+                    compile_error!("`crate` argument is not supported with codspeed_divan_compat");
                 }
                 .into();
             }
+            NameValue(MetaNameValue { path, .. }) if path.is_ident("types") => {
+                return quote! {
+                    compile_error!("`type` argument is not yet supported with codspeed_divan_compat");
+                }
+                .into();
+            }
+            NameValue(MetaNameValue { path, .. })
+                if path.is_ident("min_time")
+                    || path.is_ident("max_time")
+                    || path.is_ident("sample_size")
+                    || path.is_ident("sample_count")
+                    || path.is_ident("skip_ext_time") =>
+            {
+                // These arguments are ignored in instrumented mode
+            }
             _ => filtered_args.push(arg),
```

Comment:
if it's possible to match on the ident directly, it would be clearer to first get the `path` and then match on it. It would avoid having to repeat a big part of the pattern in each arm